### PR TITLE
Patch 10

### PR DESCRIPTION
**Pull request by AnyBananaGAME**

### CODE REVIEW

**Comment by AnyBananaGAME** [comment]
config is failing it 
smh

**Comment by AnyBananaGAME** [comment]
ok good now

File: tests.js [hunk 1]
```diff
@@ -14,6 +14,19 @@
 const ClientJoin = require("./test/ClientJoin");
 const StartServer = require("./test/StartServer");
 const TestConfigs = require("./test/TestConfigs");
+const ServerInfo = require("./src/server/ServerInfo")
+const config = ServerInfo.config;
+
+
+if(!config.offlineMode){
```
Comment (andriycraft) [inline tests.js:22]: this code breaks the tests

Comment (AnyBananaGAME) [reply tests.js:22]: didn't I make it check if config file exists

Comment (andriycraft) [reply tests.js:22]: > didn't I make it check if config file exists

u added a check that checks if the config exists before the server started wich is impossible cuz the config.yml is in the gitignore list

**Pull request merged**